test/back-references.unit.ts

```diff
@@ -1,88 +1,12 @@
 /* eslint-disable no-new */
-import { Entity } from "../src/entity";
-import { IgnoreProperty, PropertyConverter, PropertySerializationResult } from "../src/entity-serializer";
 import { Model } from "../src/model";
-import { Property } from "../src/property";
-import { isEntityType } from "../src/type";
+import { ensureChildProperties } from "./utils";
+import { IgnorePropertyConverter } from "./ignore-property-converter";
+import { InitializeBackReferencesConverter } from "./initialize-back-references-converter";
+import { IdReferencePropertyConverter } from "./id-reference-property-converter";
 
 require("../src/resource-en");
 
-class IgnorePropertyConverter extends PropertyConverter {
```

Comment:
Moved this stuff to separate files so that they could be reused across test modules.